### DISCUSSION TITLE
Here is an example of a proxy in node

### DISCUSSION THREAD

**Discussion by dk253** [chimurai/http-proxy-middleware#693] [Show and tell]
### Describe the feature you'd love to see

Howdy,

I just (finally) got this to work by setting up a proxy on my node express server.  Here is the code if you want to add it to your readme (or if you want to correct me with a better way).

```
    app.use('/api', createProxyMiddleware({
        target: 'https://blah.remotetargetexample.com/api',
        onProxyRes: function (proxyRes, req, res) {
            proxyRes.headers['Access-Control-Allow-Origin'] = '*';
        },
        changeOrigin: true,
        pathRewrite: {
            [`^/api`]: '',
        },
    }));
```

Cheers!

**Comment by chimurai** [comment]
Happy to receive a contribution for a `cors.md` recipe. (https://github.com/chimurai/http-proxy-middleware/tree/master/recipes/cors.md)

Could you please add some background information with a problem description (like the one you faced) to provide some extra context for the code example?
